airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcCtidInitializer.java

```diff
@@ -71,46 +77,28 @@ public static List<AutoCloseableIterator<AirbyteMessage>> cdcCtidIteratorsCombin
                                                                                      final JsonNode replicationSlot) {
     try {
       final JsonNode sourceConfig = database.getSourceConfig();
-      final Duration firstRecordWaitTime = PostgresUtils.getFirstRecordWaitTime(sourceConfig);
-      final Duration subsequentRecordWaitTime = PostgresUtils.getSubsequentRecordWaitTime(sourceConfig);
-      final int queueSize = PostgresUtils.getQueueSize(sourceConfig);
-      LOGGER.info("First record waiting time: {} seconds", firstRecordWaitTime.getSeconds());
-      LOGGER.info("Queue size: {}", queueSize);
 
       if (isDebugMode(sourceConfig) && !PostgresUtils.shouldFlushAfterSync(sourceConfig)) {
         throw new ConfigErrorException("WARNING: The config indicates that we are clearing the WAL while reading data. This will mutate the WAL" +
             " associated with the source being debugged and is not advised.");
       }
 
-      final PostgresDebeziumStateUtil postgresDebeziumStateUtil = new PostgresDebeziumStateUtil();
+      var initialDebeziumState = PostgresLsnMapper.makeSyntheticDebeziumState(database, sourceConfig.get(JdbcUtils.DATABASE_KEY).asText());
```

Comment:
can we type those declarations rather than using var? I think with a new API, the returned types are not as obvious as with a known one

Reply:
Yes. This wasn't a deliberate stylistic choice. When the type is not obvious, it should be declared. I was being a bit lazy, I guess.